src/__tests__/pick.test.ts

```diff
@@ -0,0 +1,130 @@
+import { plainToClass } from 'class-transformer';
```

Comment:
I have improved  test coverage a bit. I know there are some limitations to the current `pick` logic, but I always forget what they are.